superbuild/projects_modules/music-plugins.cmake

```diff
@@ -40,8 +40,8 @@ function(music_plugins_project)
 
     if (NOT USE_SYSTEM_${external_project})
 
-        set(git_url ${GITHUB_PREFIX}Inria-Asclepios/music.git)
-        set(git_tag master)
+        set(git_url ${GITHUB_PREFIX}mathildemerle/music.git)
+        set(git_tag realtime4.0.1)
```

Comment:
This needs to be removed

Reply:
yes, i'll remove that part once the PR is ok, i was using it for cross-compilation